bot/constants.py

```diff
@@ -193,14 +224,17 @@ class Roles(NamedTuple):
     muted = 277914926603829249
     owner = 267627879762755584
     verified = 352427296948486144
-    helpers = 267630620367257601
+    helpers = int(environ.get("ROLE_HELPERS", 267630620367257601))
     rockstars = 458226413825294336
     core_developers = 587606783669829632
+    events_lead = 778361735739998228
 
 
 class Tokens(NamedTuple):
     giphy = environ.get("GIPHY_TOKEN")
-    aoc_session_cookie = environ.get("AOC_SESSION_COOKIE")
+    # Public AoC cookies in environment must be in the same order as AdventOfCode.leaderboard_public_ids
+    aoc_public_session_cookies = environ.get("AOC_PUBLIC_SESSION_COOKIES", "").split(",")
+    aoc_staff_session_cookie = environ.get("AOC_STAFF_SESSION_COOKIE")
```

Comment:
These can be removed now, because they are already in `AOC_LEADERBOARDS`.

Reply:
I've removed them